internal/runners/export/config.go

```diff
@@ -0,0 +1,66 @@
+package export
+
+import (
+	"encoding/json"
+	"strings"
+
+	"github.com/ActiveState/cli/internal/captain"
+	"github.com/ActiveState/cli/internal/config"
+	"github.com/ActiveState/cli/internal/locale"
+	"github.com/ActiveState/cli/internal/output"
+)
+
+const (
+	FilterDir = "dir"
+)
+
+func RecognizedFilters() string {
+	filterLookup := []string{
+		FilterDir,
+	}
+	return strings.Join(filterLookup, ",")
+}
+
+type Config struct {
+	out output.Outputer
+}
+
+type ConfigParams struct {
+	Filter string
```

Comment:
Let's use a FlagMarshaler for this

http://github.com/ActiveState/cli/blob/b65496236a6d4273d808ae85751fca49dc327e07/internal/captain/flag.go#L9-L9

Feel free to ping me if you'd like some pointers on how to use that.